torch/testing/_internal/common_methods_invocations.py

```diff
@@ -16766,6 +16789,22 @@ def reference_flatten(input, start_dim=0, end_dim=-1):
                DecorateInfo(unittest.skip("Skipped!"), 'TestCommon', 'test_non_standard_bool_values',
                             dtypes=[torch.bool], active_if=TEST_WITH_ROCM),
            )),
+    OpInfo('nonzero_static',
+           dtypes=all_types_and_complex_and(torch.bool, torch.bfloat16, torch.float16, torch.chalf),
+           sample_inputs_func=sample_inputs_nonzero_static,
+           supports_out=False,
+           supports_autograd=False,
+           decorators=[onlyCPU],
+           skips=(
+               # Same set of skips as dynamic nonzero() above
+               DecorateInfo(unittest.expectedFailure, 'TestCommon', 'test_out'),
+               DecorateInfo(unittest.expectedFailure, 'TestCommon', 'test_out_warning'),
+               DecorateInfo(unittest.expectedFailure, 'TestJit', 'test_variant_consistency_jit'),
```

Comment:
I note that there are "Unexpected success" for this test case. After looking into it, it turns out that this test is marked expected failure for regular `nonzero()` due to a [bug](https://github.com/pytorch/pytorch/issues/67458) of passing as_tuple to it, which shouldn't affect nonzero_static(). So will remove this from the `skips` list